requirements.txt

```diff
@@ -17,7 +15,7 @@ gevent
 greenlet
 gunicorn
 localconfig
-paramiko[ssh]
+paramiko
```

Comment:
Shouldn't this be in the docker extra (with or without its ssh extra) or do we implicitly use it for something else?

Reply:
Definitely. Actually, I believe I put it there by mistake.